cmd/wit-bindgen-go/.gitignore

```diff
@@ -0,0 +1 @@
+wit-bindgen-go
```

Comment:
Can you remove this? I'd like to think about gitignore more generally.

Reply:
Might want to rebase as well!

Reply:
Ah sure! will do both!